quickstart/docker/createLocalImage.sh

```diff
@@ -1,20 +1,29 @@
 #!/usr/bin/env bash
-set -eo pipefail
+set -euo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+ZITI_BIN="${SCRIPT_DIR}/image/ziti-bin"
+
+case "${1:-}" in
```

Comment:
i had specifically left this functionality out of this script. can you add a readme.md update too?

Reply:
Added docs and sanity checks for the version override.